File: Iniciante/1071 - Soma de Impares Consecutivos I/Python/1071.py
def main():
    X = int(input())
    Y = int(input())

    print(soma_impares(X, Y))


def soma_impares(x, y):
    if x >= y:
        somador = x - 1
        controle = y
    else:
        somador = y - 1
        controle = x
    soma = 0
    while somador > controle:
        if somador % 2 != 0:
            soma += somador
        somador -= 1
    return soma


main()

File: Iniciante/1071 - Soma de Impares Consecutivos I/JavaScript/1071.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  const X = lines.shift();
  const Y = lines.shift();

  console.log(somaImpares(X, Y));
}

function somaImpares(x, y) {
  let somador;
  let controle;
  let soma = 0;
  if (x >= y) {
    somador = x - 1;
    controle = y;
  } else {
    somador = y - 1;
    controle = x;
  }

  while (somador > controle) {
    if (somador % 2 !== 0) {
      soma += somador;
    }
    somador--;
  }
  return soma;
}

main();
